src/ui/components/common/PageWrapper.jsx

```diff
@@ -25,7 +25,7 @@ export default function PageWrapper({children, showBackground, showMenu}) {
       minH="100vh"
       justifyContent="space-between"
       bg="#efefef"
-      bgImg={showBackground && "url('./images/backgrounds/landing-tile.png')"}
+      bgImg={showBackground ? "url('./images/backgrounds/landing-tile.png')" : 'none'}
```

Comment:
I changed this line because it was generating noise in the console in production and development. It's unrelated to this PR but just a small bug fix.